src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -93,8 +108,24 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     # if the user had not logged in.
     check_kube_connection(configuration)
 
+    required_node_exists = check_linux_amd64_node(configuration)
+    if not required_node_exists:
+        telemetry.set_user_fault()
+        telemetry.set_exception(fault_type=consts.Linux_Amd64_Node_Not_Exists,
+                                summary="Couldn't find any node on the kubernetes cluster with the architecture type 'amd64' and OS 'linux'")
+        raise CLIError("Couldn't connect this Kubernetes cluster to Azure Arc as it doesn't have any nodes with OS 'linux' and architecture 'amd64'. Learn more at {}".format("https://aka.ms/ArcK8sSupportedOSArchitecture"))
```

Comment:
>CLIError [](start = 14, length = 8)

Here depending on how accurate this check is, we may want to just return a warning as opposed to an error.

Reply:
Updating to warning

Reply:
https://github.com/Azure/azure-cli-extensions/pull/3001/files/a9adc9f04514fc2dd9b52f0337590769aa91a797#diff-15712b42e1d5ed0104523504aaca90e0db6c1c0a49895f113fc172cd5130b88dR515

Here is the function to check.

Reply:
The default level to be displayed to user is logger.warning, anything below that wont be displayed to user unless used the command with --debug. Whatever displayed in logger.info(),  will come in --debug mode only.

Reply:
Just warning dispalyed now